account_payment_return/models/account_journal.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 luis M. Ontalba <luis.martinez@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    default_expense_account_id = fields.Many2one(
+        comodel_name='account.account', string='Default Expense Account',
```

Comment:
string is not consistent with help.
I suggest "Default Commission Account"

Reply:
Commission is more for sales terminology. I don't think it's correct for an expense.